runtime/src/lib.rs

```diff
@@ -842,6 +897,95 @@ construct_runtime!(
     }
 );
 
+// Create the runtime by composing the FRAME pallets that were previously configured.
+#[cfg(feature = "relay")]
+construct_runtime!(
+    pub struct Runtime {
+        // Basic stuff
+        System: frame_system = 0,
+        Scheduler: pallet_scheduler = 1,
+        Preimage: pallet_preimage = 2,
+
+        Timestamp: pallet_timestamp = 3,
+        Balances: pallet_balances = 4,
+        TransactionPayment: pallet_transaction_payment = 5,
+
+        // Consensus support.
+        // Authorship must be before session in order to note author in the correct session and era
+        // for im-online and staking.
+        Authorship: pallet_authorship = 6,
+        Staking: pallet_staking = 7,
+        Offences: pallet_offences = 8,
+        Historical: pallet_session_historical = 9,
+
+        // Consensus
+        Babe: pallet_babe = 10,
+        Session: pallet_session = 11,
+        Grandpa: pallet_grandpa = 12,
+        AuthorityDiscovery: pallet_authority_discovery = 13,
+
+        // Opengov stuff
+        Treasury: pallet_treasury = 14,
+        ConvictionVoting: pallet_conviction_voting = 15,
+        Referenda: pallet_referenda = 16,
+        Origins: pallet_custom_origins = 17,
+        Whitelist: pallet_whitelist = 18,
+        VoterList: pallet_bags_list::<Instance1> = 19,
+
+        // Bounties modules.
+        Bounties: pallet_bounties = 25,
+        ChildBounties: pallet_child_bounties = 26,
+
+        // Utility modules.
+        Utility: pallet_utility = 30,
+        Multisig: pallet_multisig = 31,
+        Proxy: pallet_proxy = 32,
+
+
+        // Pallets that we know are to remove in a future. Start indices at 50 to leave room.
+        Sudo: pallet_sudo = 50,
+        ImOnline: pallet_im_online = 51,
```

Comment:
Maybe we can remove it?

Reply:
I believe we should really consider this change carefully and see if it aligns well with our use case.
Anyways, for now I'm ok with keeping this aligned with Polkadot, and I removed it.